### PR TITLE
feat(kube-monitoring): adding default pmon

### PR DESCRIPTION
**Pull request by viennaa**
services with a port metrics are getting collected now

### CODE REVIEW

File: kube-monitoring/charts/templates/pmon.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: {{ $.Release.Name }}-sd
```
Comment (ibakshay) [inline kube-monitoring/charts/templates/pmon.yaml:4]: Does "sd" stand for "service discovery" here?

File: kube-monitoring/charts/templates/tests/test-kube-monitoring-config.yaml [hunk 2]
```diff
@@ -53,6 +53,9 @@ data:
     @test "Verify creation of the prometheus-{{ .Release.Name }} statefulset" {
         verify "there is 1 statefulset named 'prometheus-{{ .Release.Name }}'"
     }
+    @test "Verify creation of the {{ .Release.Name }}-sd Podmonitor" {
```
Comment (ibakshay) [inline kube-monitoring/charts/templates/tests/test-kube-monitoring-config.yaml:56]: Adding a blank line to be consistent like other test cases. 
```suggestion

    @test "Verify creation of the {{ .Release.Name }}-sd Podmonitor" {
```

File: kube-monitoring/charts/templates/pmon.yaml (hunk 1 above)
Comment (viennaa) [reply kube-monitoring/charts/templates/pmon.yaml:4]: yes

**Comment by richardtief** [comment]
Where does this feature request come from? This will lead to multiple pod scrapes if we install another Prometheus, for example.

**Pull request merged**